docs2/site/docs/getting-started/dependency-injection.md

```diff
@@ -251,9 +251,9 @@ might look like this:
 ```csharp
 var result = await _executer.ExecuteAsync(options =>
 {
-    options.Schema = schema;
+    options.Schema = _schema;
```

Comment:
```suggestion
    options.Schema = schema;
```

Reply:
This is a DI-specific example, and as such `_schema` seemed more appropriate, so you can infer that it is pulled in via DI.  Do you not agree?

Reply:
ok